### ISSUE TITLE
orientation Portrait Only

### ISSUE THREAD

**Issue by yjj781265** [esafirm/android-image-picker#158]
Is there any way I can make ImagePicker activity Portrait Only?

**Comment by esafirm** [comment]
I think you can override the activity declaration in AndroidManifest. Haven't tried it but i think it should work

**Comment by esafirm** [comment]
@achatina it will override the current declaration in the library right?

**Comment by achatina** [comment]
@esafirm Yes, it seems so. Anyway, I think i would be great feature to add in Factory method like screenOrientation(some value), with default behavior - Changing Orientation. It will help not to overriding default declaration and may protect from some bugs

**Comment by yjj781265** [comment]
@achatina thanks I will try that.
